web/.env.example

```
@@ -1,2 +1,25 @@
+# Used by the leaderboard and results pages to fetch benchmark data.
+# Not used by the playground — playground routes call providers directly via Vercel env vars below.
 NEXT_PUBLIC_API_URL=http://localhost:8000
+# ---------------------------------------------------------------------------
+# PostHog analytics
+# Add real values locally and in Vercel before enabling event capture.
+# Optional local debug: set to `true` or append `?posthog_debug=1` to a URL
+# to log captured PostHog events in the browser console.
+# ---------------------------------------------------------------------------
+NEXT_PUBLIC_POSTHOG_TOKEN=
+NEXT_PUBLIC_POSTHOG_HOST=https://us.i.posthog.com
+NEXT_PUBLIC_POSTHOG_DEBUG=false #intentionally disabled for time being
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_POSTHOG_DEBUG key should go before the NEXT_PUBLIC_POSTHOG_HOST key

(UnorderedKey)